themes/theme-b2b/src/components/TextArea.styles.ts

```diff
@@ -18,6 +18,6 @@ export const TextArea: ThemeComponent<'TextArea'> = cva([
   inputDisabled(),
   inputError('group-error/field'),
   inputSpacing,
-  'text-text-body font-body leading-8',
-  'read-only:focus:border-border-light read-only:focus:shadow-none',
+  'text-text-base font-body leading-8',
+  'read-only:focus:border-border-base read-only:focus:shadow-none',
```

Comment:
This one I don't really understand. When focused, the border of the text area turns blue, but this color here is gray[400]. Maybe I'm misunderstanding something?

Reply:
Hmm... It should work when read-only is true 🤔 thats why read-only is set before focus

Reply:
@tirado-rx I fixed it, should be work now :)